### ISSUE TITLE
Fails to install on python 3.5 due to copyright symbol in LICENSE file

### ISSUE THREAD

**Issue by amishHammer** [bfontaine/freesms#1]
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 24, in <module>
    license=open('LICENSE', 'r').read(),
  File "/usr/local/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 10: ordinal not in range(128)

**Comment by amishHammer** [comment]
Also the - is not an ascii char

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 24, in <module>
    license=open('LICENSE', 'r').read(),
  File "/usr/local/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 19: ordinal not in range(128)

**Comment by bfontaine** [comment]
I’m able to install it without issue:

```
$ virtualenv --python python3 venv
Running virtualenv with interpreter /usr/local/bin/python3
Using base prefix '/usr/local/Cellar/python3/3.5.2_3/Frameworks/Python.framework/Versions/3.5'
New python executable in /private/tmp/venv/bin/python3.5
Not overwriting existing python script /private/tmp/venv/bin/python (you must use /private/tmp/venv/bin/python3.5)
Installing setuptools, pip, wheel...done.

# From Pypi
$ ./venv/bin/pip install freesms
Collecting freesms
  Downloading freesms-0.1.0.tar.gz
Building wheels for collected packages: freesms
  Running setup.py bdist_wheel for freesms ... done
  Stored in directory: /Users/baptiste/Library/Caches/pip/wheels/3a/62/aa/92e2044e58016261bd9828d88818f7e6125cba1c10834cf2dc
Successfully built freesms
Installing collected packages: freesms
Successfully installed freesms-0.1.0

$ ./venv/bin/pip uninstall --yes freesms
Uninstalling freesms-0.1.0:
  Successfully uninstalled freesms-0.1.0

# From this repo
$ git clone https://github.com/bfontaine/freesms
Cloning into 'freesms'...
remote: Counting objects: 47, done.
remote: Total 47 (delta 0), reused 0 (delta 0), pack-reused 47
Unpacking objects: 100% (47/47), done.
$ ./venv/bin/pip install ./freesms
Processing ./freesms
Installing collected packages: freesms
  Running setup.py install for freesms ... done
Successfully installed freesms-0.1.0
```

**Comment by amishHammer** [comment]
Please try with the LANG env var set to C.

Thanks

**Comment by bfontaine** [comment]
Thanks; I can now reproduce:

```
$ LANG=C ./venv/bin/pip install ./freesms
Processing ./freesms
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/var/folders/0d/mmrjgs1n0zdfdr386xlnx61h0000gn/T/pip-viu9q7l9-build/setup.py", line 24, in <module>
        license=open('LICENSE', 'r').read(),
      File "/private/tmp/venv/bin/../lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 10: ordinal not in range(128)

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /var/folders/0d/mmrjgs1n0zdfdr386xlnx61h0000gn/T/pip-viu9q7l9-build/
```

**Comment by bfontaine** [comment]
Fixed; I just released 0.1.1. Thanks for the report!
